content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -23,8 +23,8 @@ deploy Kubeflow on Amazon Web Services (AWS).
 
 You do not need to have an existing Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster. The deployment process will create a cluster for you.
 
-The install tools uses `eksctl` command, and doesn't support `--profile` option, now.
-So please use `aws sts assume-role` commands with [this document](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html) if you use switch role.
+The installation tool uses the `eksctl` command and doesn't support  the `--profile` [option]().
```

Comment:
There's a rendering problem on the page, due to the brackets without a link:
https://deploy-preview-947--competent-brattain-de2d6d.netlify.com/docs/aws/deploy/install-kubeflow/

Please change this:

```
and doesn't support  the `--profile` [option]().
```

to this:

```
but doesn't support  the `--profile` option in that command.
```

I've changed the wording to make it clear where the profile option exists. It wasn't clear to me, from the current wording, whether the profile option was in kfctl or eksctl or even somewhere else.

Reply:
Thank you. I fixed it w.r.t. your comment.
f71b9f6